darglint/integrity_checker.py

```diff
@@ -93,8 +96,8 @@ def run_checks(self, function):
 
         """
         self.function = function
-        if function.docstring is not None:
-            try:
+        try:
```

Comment:
Could you move the try-block back inside of the conditional?  The else-branch can't raise a ParserException, so there's no need for it to be included in the try-block.